src/pages/Admin/Layout.tsx

```diff
@@ -9,7 +9,7 @@ export default function Layout({ linkGroups }: { linkGroups: LinkGroup[] }) {
       <SidebarOpener className="md:hidden" linkGroups={linkGroups} />
       <Sidebar className="max-md:hidden" linkGroups={linkGroups} />
       {/** views */}
-      <div className="p-10">
+      <div className="padded-container p-10">
```

Comment:
`padded-container` shouldn't be applied here as other admin routes doesn't need this 

if want to center `WithdrawForm` , `/withdraws` already has `grid` layout

Reply:
Added it keeping zoomed out webapp in somewhat good proportions, but can be removed:
- with `padded-container`:
![image](https://user-images.githubusercontent.com/19427053/223168977-6324a087-1ba8-48e1-ba0f-93ce8ad8ccb0.png)

- without `padded-container`:
![image](https://user-images.githubusercontent.com/19427053/223168621-4717e8a7-bd62-4d86-9fe2-d14d90650cf9.png)

**@SovereignAndrey do we even EVER want to handle the too zoomed out webapp UI (e.g. zooming out to <50%)?**

Reply:
@misicnenad @ap-justin We can leave `padded-container` class off there, esp if none of the other admin areas use it currently and address that as a whole admin area (as it does look much better with it on imo).

Reply:
@misicnenad, @SovereignAndrey,  I see now the use case of this padded container here, but `padded-container` is 
```
xl:container xl:mx-auto px-5
```
`px-5` would be applied on `mobile` views where it's supposed to be just `px-4`

could just add 
```
xl:container, xl:mx-auto
```

also `padded-container` is  intended to be used on standalone pages not sections to align with `header`. For sections, you need to offset the width of the sidebar

<img width="339" alt="image" src="https://user-images.githubusercontent.com/89639563/223621588-b803e092-ce4e-4530-a43a-297366c76d04.png">

Reply:
Resolving this as the problem was addressed in PR https://github.com/AngelProtocolFinance/angelprotocol-web-app/pull/1879, now part of this PR (after rebase).